app/example_app/announcement_board.py

```diff
@@ -0,0 +1,84 @@
+from flask import render_template, session, redirect, url_for, current_app, request
+from .. import db
+from ..models import User
+from ..models import Announcement
+from . import example_app
+from .forms import NameForm
+from datetime import date
+
+@example_app.route('/announcement_board')
+def home():
+    return render_template("announcement_board/readerOrWorker.html")
+
+@example_app.route('/announcement_board/reader')
+def reader():
+    return render_template("announcement_board/reader.html")
+
+@example_app.route('/announcement_board/worker')
+def worker():
+    return render_template("announcement_board/worker.html")
+
+
+@example_app.route('/announcement_board/worker/add_announcement', methods=['GET', 'POST'])
```

Comment:
Nazwa może być krótsza. Aplikacja rozróżnia czy wysyłacie GET, czy POST dlatego add jest niepotrzebne.
```suggestion
@example_app.route('/announcement/worker/', methods=['GET', 'POST'])
```